### PR TITLE
Fix #1816 (Very inefficient managegroups.php code)

### LINKED ISSUE #1816: Very inefficient managegroups.php code...
https://github.com/mybb/mybb/blob/feature/managegroup.php#L376 - that's quite bad, one query is executed per every group member while the leaders could be simply added to an array in this loop: https://github.com/mybb/mybb/blob/feature/managegroup.php#L307 and then checked with `in_array()`.

### PR DESCRIPTION
**Pull request by Krytic**
Fix #1816 => Very inefficient managegroups.php code

### CODE REVIEW

**Comment by Krytic** [comment]
Pushed a quick fix where the "(Leader)" string wasn't being displayed.

File: managegroup.php [hunk 1]
```diff
@@ -375,11 +380,10 @@
 		{
 			$email = '';
 		}
-		$query1 = $db->simple_select("groupleaders", "uid", "uid='{$user['uid']}' AND gid='{$gid}'");
-		$isleader = $db->fetch_array($query1);
+		$isleader = in_array($user['uid'], $leaders_array);
```
Comment (Destroy666x) [inline managegroup.php]: No need for the variable, you can simply check it in the conditional

Comment (Krytic) [inline managegroup.php]: Thanks @Destroy666x -- pushing that commit now

**Comment by Destroy666x** [comment]
Haven't noticed any problem now, thanks for the fix.

**Pull request merged**